staging/publishing/rules.yaml

```diff
@@ -1,4 +1,7 @@
 recursive-delete-patterns:
+# TODO: remove bazel related files after we stop publishing branches with
```

Comment:
@nikhita @dims 

Since it will be a few releases before we can do this, leaving a TODO

Reply:
> Since it will be a few releases before we can do this, leaving a TODO

:+1: thanks!